.gitattributesXX

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
W

Reply:
A leftover due to weird things happening during the move operation. I removed it and rebased...